core/templates/services/exploration-features-backend-api.service.ts

```diff
@@ -46,7 +46,7 @@ export class ExplorationFeaturesBackendApiService {
   fetchExplorationFeatures(
       explorationId: string): Promise<ExplorationFeatures> {
     return this.http.get<ExplorationFeaturesBackendDict>(
-      this.urlInterpolationService.interpolateUrl(
+      <string> this.urlInterpolationService.interpolateUrl(
```

Comment:
```suggestion
      this.urlInterpolationService.interpolateUrl<string>(
```

Are we expecting it to be like this^?